packs/data/feats.db/ongoing-strategy.json

```diff
@@ -27,7 +27,7 @@
         },
         "rules": [
             {
-                "category": "precision",
+                "damageCategory": "precision",
```

Comment:
Should just be category.

Reply:
`damageCategory` for FlatModifier

Reply:
Is there a reason it's different for flat modifier than damage dice?

Reply:
Because it is implicit on damage dice and not so on flat modifiers.